src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -515,7 +515,7 @@ def DeviceAvailableCallback(device, err):
                 deviceAvailableCV.wait()
 
         if returnDevice.value is None:
-            raise self._ChipStack.ErrorToException(CHIP_ERROR_INTERNAL)
+            raise RuntimeError("Failed to get connected device!")
```

Comment:
It appears that this made assumptions about constants being available which are not. However, I would argue that the Runtime error here may be more confusion. I feel like many of these Python API methods would benefit from a more specific MatterException/ChipException hierarchy that has additional metadata about failures